net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+         export OSCX_FLATPAK_INCLUDES="-I/app/include/tirpc" && \
+         make -j ${FLATPAK_BUILDER_N_JOBS} prefix=/app \
+         oscx
+  - |
+    make -j ${FLATPAK_BUILDER_N_JOBS} prefix=/app LUA_LIBS="-L/app/lib -llua" \
+         pdlua
+  # now let's install them
+  - |
+    make -j ${FLATPAK_BUILDER_N_JOBS} DESTDIR=/ prefix=/app \
+         adaptive_install arraysize_install autotune_install bassemu_install \
+         boids_install bsaylor_install \
+         comport_install creb_install cxc_install cyclone_install \
+         disis_install earplug_install ekext_install ext13_install fftease_install \
+         flatgui_install fluid_install freeverb_install gem_install ggee_install \
+         hcs_install iem_ambi_install iem_bin_ambi_install iemlib_install \
+         iemguts_install iem_adaptfilt_install iemmatrix_install iemxmlrpc_install \
+         iem_delay_install iem_roomsim_install iem_spec2_install iem_tab_install \
+         iemgui_install jasch_lib_install loaders-libdir_install lyonpotpourri_install \
+         mapping_install markex_install maxlib_install mjlib_install \
+         moocow_install moonlib_install motex_install mrpeach_install oscx_install \
+         pan_install pdcontainer_install pddp_install pdlua_install pdogg_install \
+         plugin_install pmpd_install rjlib_install sigpack_install smlib_install \
+         tof_install unauthorized_install vbap_install windowing_install zexy_install
+
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231012'
+    commit: 27b66ed5efd4de7cb0fd7ad33cf46c60cb8201fc
+
+- name: pd-abstractions
+  buildsystem: simple
+  subdir: abstractions
+  build-commands:
+  - make -j ${FLATPAK_BUILDER_N_JOBS} DESTDIR=/ prefix=/app install
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231012'
+    commit: 27b66ed5efd4de7cb0fd7ad33cf46c60cb8201fc
+
+- name: pd-l2ork-integration
+  buildsystem: simple
+  build-commands:
+  # These commands run in the 'subdir' that we set, but the sources appear at
+  # the top level hence the `../..` everywhere.
+  - mkdir -p /app/lib/pd-l2ork/bin/nw
+  - tar -x -f nwjs.tar.gz -C /app/lib/pd-l2ork/bin/nw --strip-components=1
+  # wrapper script
+  - install -m 755 pd-wrapper.sh /app/bin
+  # .desktop file (so we appear in list of applications)
+  - install -d /app/share/applications/
+  - install -m 644 net.pdl2ork.PdL2Ork.desktop /app/share/applications/
+  # AppStream metadata (for app store / package manager)
+  - install -d /app/share/metainfo/
+  - install -m 644 net.pdl2ork.PdL2Ork.metainfo.xml /app/share/metainfo
+  # icon (for everyone)
+  - install -d /app/share/icons/hicolor/scalable/apps
+  - install -m 644 net.pdl2ork.PdL2Ork.svg /app/share/icons/hicolor/scalable/apps/
+  # add mime type to .pd files
+  - install -d /app/share/mime/packages/
+  - install -m 644 packages/linux_make/pd-l2ork.xml /app/share/mime/packages/net.pdl2ork.PdL2Ork.xml
```

Comment:
```suggestion
  - install -Dm755 pd-wrapper.sh /app/bin
  # .desktop file (so we appear in list of applications)
  - install -Dm644 net.pdl2ork.PdL2Ork.desktop -t /app/share/applications/
  # AppStream metadata (for app store / package manager)
  - install -Dm644 net.pdl2ork.PdL2Ork.metainfo.xml -t /app/share/metainfo
  # icon (for everyone)
  - install -Dm644 net.pdl2ork.PdL2Ork.svg -t /app/share/icons/hicolor/scalable/apps/
  # add mime type to .pd files
  - install -Dm644 packages/linux_make/pd-l2ork.xml /app/share/mime/packages/net.pdl2ork.PdL2Ork.xml
```

Reply:
How do I pull this commit into my branch once I merge it as part of this pull request?

Reply:
I don't understand your question.